src/libpriv/rpmostree-unpacker.c

```diff
@@ -750,13 +750,6 @@ import_rpm_to_repo (RpmOstreeUnpacker *self,
       if (!ostree_repo_write_dfd_to_mtree (repo, tmpdir.fd, ".", mtree, modifier,
                                            cancellable, error))
         return FALSE;
-
-      /* check if any of the cbs set an error */
```

Comment:
Ah, tricksy.  I should have run through annotate :notebook:.  Mmm, not sure about switching up the filter just for coverity; I'd vote we add a bit to suppress that check.  Would need to find the docs for that.